internal/maincmd/maincmd.go

```diff
@@ -1,3 +1,7 @@
+// Package maincmd implements a subset of the '$ rsync' CLI surface, namely that it can:
+// - serve as a server daemon over TCP or SSH (via SSH session stdin/stdout)
```

Comment:
If you put 3 space characters in front of the `-`, godoc will render this list correctly (with one item per line)

Reply:
Sure, thanks. You always learn something new :)